aws-cli-node/aws-cli-node:16.6.1-stretch.Dockerfile

```
@@ -0,0 +1,9 @@
+# docker build . -f aws-cli-node:16.6.1-stretch.Dockerfile -t justincasetech/aws-cli-node:16.6.1-stretch
+# docker push justincasetech/aws-cli-node:16.6.1-stretch
+FROM node:16.6.1-stretch
```

LTSに突入したバージョンにしてほしいです

```suggestion
FROM node:16.14.0-stretch
```